packages/uniorg-parse/src/parse-options.ts

```diff
@@ -25,6 +25,10 @@ export interface ParseOptions {
    * matching. Corresponds to `org-match-sexp-depth` in Emacs.
    */
   matchSexpDepth: number;
+  /**
+   * Add begin/end properties to primitive nodes
+   */
+  positions: boolean;
```

Comment:
To make it compatible with other unified parsers:
```suggestion
  position: boolean;
```